src/models/clients.js

```diff
@@ -22,6 +22,9 @@ module.exports = (sequelize, DataTypes) => {
       defaultValue: () => {
         return randomBytes(64).toString('hex');
       }
+    },
+    redirectUrl: {
```

Comment:
nit: tell me what you think, I would use "authRedirectUrl" to make it clear, when looking at the database, that this is for the authentication process.